scss/c2s/c2s.scss

```diff
@@ -1,6 +1,13 @@
 @import "betahealth/nhsuk";
 
 /*
+   Colours.
+*/
+$hiviz: #ffca55;
+$blue: #0072c6;
+$callout-border-blue: #40ADD7;
```

Comment:
I can't see the usage of `$callout-border-blue` or `$callout-blue`. Should the be used at https://github.com/nhsuk/profiles/pull/30/files#diff-fbb593c07f7b6259a85d6bb2324072c7R108 ?